test.sh

```diff
@@ -61,6 +61,16 @@ $RUN rm -rf /tmp/osbs-client
 $RUN git clone https://github.com/projectatomic/osbs-client /tmp/osbs-client
 $RUN $BUILDDEP -y /tmp/osbs-client/osbs-client.spec
 $RUN $PIP install --upgrade --no-deps --force-reinstall git+https://github.com/projectatomic/osbs-client
+
+# Install dockerfile-parse in the same way as osbs-client
+$RUN rm -rf /tmp/dockerfile-parse
+$RUN git clone https://github.com/containerbuildsystem/dockerfile-parse \
```

Comment:
could we set `--depth <depth>` here to 1?

Reply:
I'm also not a big fan of cloning that repo, do we really need it? Cannot we just use pip with dependencies?

Reply:
Ok, not cloning dfp. Note that this unfortunately does not work for osbs-client, it seems that some of the `BuildRequires` dependencies are actually required for building (other than `devel` and `setuptools`).